### PR TITLE
chapter11

### PR DESCRIPTION
**Pull request by xtrxllx**

### CODE REVIEW

**Comment by xtrxllx** [comment]
ready

File: app/assets/stylesheets/account_activations.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the AccountActivations controller here.
```
Comment (manhnt-0960) [inline app/assets/stylesheets/account_activations.scss]: delete this empty file

File: app/controllers/account_activations_controller.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class AccountActivationsController < ApplicationController
+    def edit
```
Comment (manhnt-0960) [inline app/controllers/account_activations_controller.rb]: indent 2 spaces

File: app/controllers/account_activations_controller.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+        log_in user
+        flash[:success] = t "user.new.activate_success"
+        redirect_to user
+      else
+        flash[:danger] = t "user.new.activate_fail"
+        redirect_to root_url
+      end
+    end
+  end
+
```
Comment (manhnt-0960) [inline app/controllers/account_activations_controller.rb:15]: check end line

File: app/helpers/account_activations_helper.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+module AccountActivationsHelper
```
Comment (manhnt-0960) [inline app/helpers/account_activations_helper.rb]: delete this empty file

**Comment by xtrxllx** [comment]
ready

**Comment by manhnt-0960** [comment]
merged

**Pull request merged**